docs/site/FAQ.md

```diff
@@ -17,23 +19,75 @@ summary: LoopBack 4 is a completely new framework, also known as LoopBack-Next.
 
 See [Crafting LoopBack 4](Crafting-LoopBack-4.md) for more details.
 
-### Where are the tutorials?
+### Is LoopBack 4 a REST framework?
 
-See [Examples](Examples.md) and [Tutorials](Tutorials.md).
+**Yes and no.** Unlike other REST frameworks, LoopBack 4's scope spans beyond
+creating REST APIs, and the individual Node.js packages are designed with this
+in mind. This leads to increased flexibility for more complex use-cases and
+enables reuse of concepts across different projects.
 
-### What features are planned?
+LoopBack 4's focus is still REST APIs and will continue to innovate in improving
+the API developer experience.
 
-Our roadmap is captured in
-[here](https://github.com/strongloop/loopback-next/labels/roadmap). You can also
-check out our
-[monthly milestone plans](https://github.com/strongloop/loopback-next/labels/Monthly%20Milestone).
+See
+[Can non-REST projects utilize LoopBack 4?](#can-non-rest-projects-utilize-loopback-4)
+for details on how to leverage LoopBack in other projects.
 
-For LoopBack 3 users, here is the list of LB3 feature parity we wish to be
-implemented in the future:
-https://github.com/strongloop/loopback-next/issues/1920.
+### Why introduce so many new concepts?
 
-Do you have some other features in mind? Add your feature requests at
-[loopback-next/issues/new](https://github.com/strongloop/loopback-next/issues/new).
+At first glance, concepts such as Inversion of Control, Dependency Injection,
+Services and Providers may seem daunting when coming from other typical Node.js
+projects. However, these have been well-defined and battle-tested in many other
+programming languages such as C# and Java. Furthermore, they have been rapidly
+adopted by other Node.js project such as Angular.
+
+These concepts improve code readability and predictability by promoting use of
+SOLID principles, and by doing so, aid in code maintainability in the long term.
+
+### Can non-REST projects utilize LoopBack 4?
+
+**Yes they can!** LoopBack 4 was designed to be flexible to support non-REST
+projects. Here's a few examples:
+
+- `@loopback/context`
+
+  Dependency injection framework. Can be used to in any Node.js project to
+  improve code maintainability.
+
+- `@loopback/core`
+
+  Builds on top of `@loopback/context` and introduces higher-level concepts such
+  as Services to further improve code maintainability.
+
+- `@loopback/metadata`
+
+  Create and inspect TypeScript decorators to define, merge and inspect
+  metadata.
+
+- `@loopback/repository`
+
+  Cross-database ORM and ODM framework. Can be used to create cross-database
+  designs, queries and relations.
+
+- `@loopback/filter`
+
+  Contains LoopBack filter typings and builders. Can be used to build queries on
+  LoopBack API consumers.
+
```

Comment:
Can we mention recently introduced GraphQL support? See https://github.com/strongloop/loopback-next/tree/master/extensions/graphql